src/lavinmq/http/controller/exchanges.cr

```diff
@@ -71,6 +71,8 @@ module LavinMQ
               context.response.status_code = 204
             elsif name.starts_with? "amq."
               bad_request(context, "Not allowed to use the amq. prefix")
+            elsif name.size > 255
```

Comment:
always use `String#bytesize`, it's different from `String#size` when regarding multibyte characters.